### PR TITLE
Aman memory-case-association activity

### PR DESCRIPTION
**Pull request by gupta2140**
Added copyrights, manual of the activity, image and sound dataset for the alphabets, Activity icon yet to be added.

### CODE REVIEW

File: src/activities/memory-case-association/ActivityInfo.qml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import GCompris 1.0
+
+ActivityInfo {
+  name: "memory-case-association/MemoryCaseAssociation.qml"
+  difficulty: 2
+  icon: "memory-case-association/memory-case-association.svg"
+  author: "Aman Kumar Gupta &lt;gupta2140@gmail.com&gt;"
+  demo: false
+  //: Activity title
+  title: "CaseAssociation memory game"
```
Comment (petitlapin) [inline src/activities/memory-case-association/ActivityInfo.qml]: qsTr

File: src/activities/memory-case-association/ActivityInfo.qml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  //intro: "Match the alphabet with its upper/lower case alphabet."
+  //: Help goal
+  goal: qsTr("Learning lower and upper case alphabets, memory.")
+  //: Help prerequisite
+  prerequisite: qsTr("Knowing alphabets")
+  //: Help manual
+  manual: qsTr("You can see some cards, but you can't see what's on the other side of them. Each card is hiding the lower/uppercase of an alphabet, and you have to associate all the upper case letters with its lower case and vice versa.")
+  credit: ""
+  section: "reading"
+}
```
Comment (petitlapin) [inline src/activities/memory-case-association/ActivityInfo.qml]: missing createdInVersion

File: src/activities/memory-case-association/ActivityInfo.qml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+ * along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+import GCompris 1.0
+
+ActivityInfo {
+  name: "memory-case-association/MemoryCaseAssociation.qml"
+  difficulty: 2
+  icon: "memory-case-association/memory-case-association.svg"
+  author: "Aman Kumar Gupta &lt;gupta2140@gmail.com&gt;"
+  demo: false
```
Comment (petitlapin) [inline src/activities/memory-case-association/ActivityInfo.qml]: should be true

File: src/activities/memory-case-association/dataset.js [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+
+            [GCompris.ApplicationInfo.getAudioFilePath("voices-$CA/$LOCALE/alphabet/" + Core.getSoundFilenamForChar('Z')),
+             GCompris.ApplicationInfo.getAudioFilePath("voices-$CA/$LOCALE/alphabet/" + Core.getSoundFilenamForChar('Z'))]
+        ]
+
+var memory_cards = [
+            { // Level 1
+                columns: 3,
+                rows: 2,
+                images: images.slice(0,6),
```
Comment (petitlapin) [inline src/activities/memory-case-association/dataset.js]: add a space after the comma

File: src/activities/memory-case-association/MemoryCaseAssociation.qml [hunk 5]
```diff
@@ -0,0 +1,26 @@
+/* GCompris - memory-case-association.qml
```
Comment (divyam3897) [inline src/activities/memory-case-association/MemoryCaseAssociation.qml]: MemoryCaseAssociation.qml?

File: src/activities/memory-case-association/dataset.js [hunk 6]
```diff
@@ -0,0 +1,163 @@
+/* GCompris - memory-case-association.js
```
Comment (divyam3897) [inline src/activities/memory-case-association/dataset.js]: dataset.js?

**Comment by gupta2140** [comment]
@divyam3897  hi, this activity is actually about matching the lower case alphabets card with it's upper case card. Here's the description for it: http://gcompris.net/wiki/Ideas_for_activities#Case_association. But if it's  better to first present Upper case letters in initial levels, followed by lower case letters and then in last levels, mix them, please let me know.  :)

**Comment by gupta2140** [comment]
And regarding translation, I'm not sure as Anintim or Johnny never mentioned about it for this activity.

**Comment by Animtim** [comment]
About "translating" this activity: I've read the discussion on irc, and I also think this activity should be kept simple, and only show simple roman letters.

It looks good to go for me, but a few things should be fixed outside of the activity:
-M and W are wider than the cards (need to check and fix the way images are sized in memory)
-a few letters have weird rendering on some angles (like c and e ), I will fix that personally in master.

**Comment by petitlapin** [comment]
Merged in https://commits.kde.org/gcompris/81fe99e145a4552556721f1aeb150fa3dd398d07
Thank you!
Johnny

**Pull request closed without merging**